Package.swift

```diff
@@ -0,0 +1,3 @@
+import PackageDescription
+
+let package = Package(name: "XCTest")
```

Comment:
This probably wouldn't be the package name desired for both OS X and Linux. We can update this if we ever want to merge this PR.